proto/viam/app/v1/app.proto

```diff
@@ -250,10 +250,16 @@ message GetRobotPartResponse {
   string config_json = 2;
 }
 
+message PageToken {
+  string log_id = 1;
+}
+
 message GetRobotPartLogsRequest {
   string id = 1;
   bool errors_only = 2;
   optional string filter = 3;
+  optional int64 offset = 4;
+  optional PageToken page_token = 5;
```

Comment:
It can just be a string type instead of PageToken message.

Reply:
The reason to encode this is because the token should be opaque to the client. The client does not care and show not know if we use the ObjectID or anything else. It just needs a token which is a string to be passed back and forth.